pkg/controllers/provisioner/v1alpha1/scheduling/types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduling
+
+import v1 "k8s.io/api/core/v1"
+
+// Scheduler assigns incoming pods to existing nodes.
+type Scheduler interface {
+	// Schedule a set of pods on existing nodes.
+	Schedule() ([]*v1.Pod, error)
```

Comment:
Shouldn't this be taking a slice of pods as an argument?

Reply:
Good call yeah. For now, this is a no-op since we're skipping scheduling. Happy to fix, or we can clean up next time someone touches it.